ReactAndroid/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.facebook.react">
 
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.VIBRATE"/>
```

Comment:
This means any app using React Native will now require the Vibrate permission. I would much prefer shipping this as a custom module although we're still working on a system to do that, hold tight: https://github.com/facebook/react-native/issues/2679

See also https://facebook.github.io/react-native/docs/known-issues.html#publishing-modules-on-android